<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teeth_Calculator</class>
 <widget class="QWidget" name="Teeth_Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>829</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="select_folder">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Folder</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="foldername">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="crop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>580</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="white_s">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Canine Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="white_e">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>670</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Canine End</string>
   </property>
  </widget>
  <widget class="QPushButton" name="black_e">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>750</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cavity End</string>
   </property>
  </widget>
  <widget class="QPushButton" name="black_s">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>710</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cavity Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_slice">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>670</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Slice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>790</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="white_pen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>580</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Mask</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="image3d">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string>3D Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ratio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>650</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="croped_result">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>650</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>521</width>
     <height>521</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>519</width>
      <height>519</height>
     </rect>
    </property>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>image</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="transparent">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="resultcanine">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>730</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Size Reset</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="gap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>750</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="slicegap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>750</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Slice Gap(mm)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shade_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>580</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Shade</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="slicetable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>630</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Slice List</string>
    </property>
   </column>
  </widget>
  <widget class="QSpinBox" name="whitestartbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>630</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="whiteendbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>670</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="blackstartbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>710</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="blackendbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>750</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="prev">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="image_control">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Control</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_slice">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>710</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="currentindex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_point">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>790</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="caninevolume">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>730</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Canine Volume :</string>
   </property>
  </widget>
  <widget class="QLabel" name="cavityvolume">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>760</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cavity Volume  :</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="resultcavity">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>760</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="resultratio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>790</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="volumeratio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>790</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume Ratio    :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="slicebox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>630</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
